lib/flags.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+class Flags {
+  static from(...args) {
+    const values = args;
+    class FlagsClass {
+      static from(...args) {
+        return new FlagsClass(...args);
+      }
+
+      constructor(...args) {
+        this.values = new Map(values.map(v => [v, 0]));
+        args.forEach(arg => this.set(arg));
+      }
+
+      has(key) {
+        return this.values.has(key);
+      }
+
+      get(key) {
```

Comment:
I don't think there is any sense in this method?

Reply:
We may need to set bit at a specific position.

Reply:
This is not setting a bit, at least it shouldn't. It's a `get` method